### ISSUE TITLE
Vafnar's weekly item updates 2/15/17

### ISSUE THREAD

**Issue by Vafnar** [AngelArenaAllstars/suggestions#181]
Won't be on for a few days, pls feedback

Great Titanium Greatsword of Greatness
Build from: Battlefury, Dadulus, Recipe
Bonuses: +175 damage, +90% cleave damage,  +5 all stats
Negatives: -50 range (melee) -150 range (ranged heroes) -60 attack speed
Active: The next strike deals 300% critical damage and you are Fatigued for 10 seconds. 30 second cooldown
Fatigued: -33% attack speed 1.5x healing
TL;DR: Gain loads of damage and cleave at the cost of range and attack speed. If this was to go into the game, rapier would need more damage and the active is there for quick bonus damage at the cost of less damage for a few seconds

Flask of Restoration
Consumable like Bottle, holds 5 charges.
Active: Gain 100 health and mana instantly
TL;DR: Alternative to bottle if you wish to have regen while in a fight and bottle is over time health while this is instant, they are supposed to have the same amount of regen but this has a longer cooldown in between uses. Bottle for farming, Flask for fighting.

Raging Axe of Throwing
Bonuses: +3% damage for melee, +1% damage for ranged +2 main attribute
Active: Throw the axe to deal 5% of your enemies health as damage instantly
TL;DR: This is supposed to be a way for brutes to get more damage and a way early on to get quick damage. When one is bought there is a cooldown on when another can be bought, like a ward or raindrop.

Robin Hood’s Quiver
Builds from: TBD
Bonuses: +25% attack speed (ranged)  +10% attack speed (melee) +50% attack range (ranged only) +10% main attribute
Negatives: +7% miss rate, -10% damage
Active: Triple your attack speed for 5 seconds and gain Fatigue for 7 seconds once the duration is over, this will break attack speed limit.
Fatigued: -33% attack speed and 1.5x healing
TL;DR: Attack speed at the cost of consistent damage

Pulsar
Passive: Every 4 seconds you gain +25% movespeed and attack speed for 1 second
Every few seconds sends a pulse out that will deal small damage and knock enemies back a short distance
TL;DR: This is to be an area denial tool that can also cut enemies off. This is to be a strong utility tool that can be used in many different situation. This can be upgraded

Blunt Saxophone
Bonuses: +10% crit chance, +30 damage, +5 to all stats
Aura: +10% movespeed, +3 to all stats
Active: Channel for 3 seconds Heal your team for 25% health and give them 30% damage, movespeed, and attack speed. This will cost 75% of your total mana over the channel
Tl;DR: ( ͡° ͜ʖ ͡°) cue the sax 

War Banner
Passive: -33% AS, -33% Damage, +5 to all stats
Aura: +15% AS, 15% damage
Active: Place a banner onto the ground making a large AoE aura providing your team with a large boost to damage and attack speed.
TL;DR: The holder of this will have reduced damage output while supplying the team with that which was lost. The wielder will not receive the damage buff or the active buff.

HamAxe
Toggle: Switch between a hammer and an axe
Axe: Gain DoT damage on auto attacks, and a cleave. Gain Counter Helix Does not stack on Axe
Hammer: Gain +70% damage and auto attacking someone causes damage to every non-hero unit in an AoE. Gain +42% bash chance
Passive: Gain +33% damage and +6 in all stats
TL;DR: The toggle is meant for different situations, Axe is for farming or poking, Hammer is for fighting and CC (crowd control).

Slime Shovel of Doom
Passive: Auto attacks slow for 50% target's movespeed
Aura: 66% increased regen rate.
Active: Create a wall of slime in front of you that puts all enemies in a Stasis while in the wall targets can’t be damaged nor can they regen. The walls have their own separate HP pool and can be taken down after a lot of damage. Anyone with a shovel can walk through no problem
TL;DR: Think of Mei ice wall that can freeze people in place

The Death Sabre
Passive: 66% increased damage, attack speed, and movespeed. Every hero kill grants +15 damage
Negatives: -50% health and mana
TL;DR: The glass cannon of items

The Mad God’s Armour
Passive: +45 armour
Negatives: -100% magic resistance Cannot walk in a straight line
TL;DR: Armour

**Comment by VoidsKeeper** [comment]
Lol , I'm guessing the numbers are just for show :P ?
So :

(1)
I don't really see why people would buy this(it doesn't give too much of a positive bonus in exchange for the negative affects). Only if someone with high-damage anyway  (Earthshaker , Tiny, PA, etc.)(could get abused by Sniper's aghs ult). Also , by the "cleave for ranged" do you mean the "holy book" effect from AAR? 

(2)
I think this has been reviewed before. My opinion is still the same - it's nicer for me to have a bottle (buutt , that's just me)

(3)
Even though the numbers don't mean anything atm , I still believe stuff shouldn't be %-wise: First - the would do almost no damage at the beginning of the game ( ~50-60hp before lvl6 :/ ) and second - it wouldn't be bought at the end of the game anyway , since most heroes will be 5-6 slotted  (hopefully .-. )
The item idea , as a concept seems interesting (to throw a consumable,that is )

(4)
We don't want to mess a lot with AS . And again - I don't think this should be %-wise

(5)
This either needs to be well tinkered with (balance issues , I guess...) or somehow changed (dunno how though. And again - AS :/ )

(6)
Yes, please (^_~) . 

(7)
This is interesting , but needs well balancing. Should bring it up on the next meeting (do it Vafnar ;) )

(8)
I think this would be too delicious for trolls , but could work?

(9)
This needs a max limit for bonus damage or make the accumulated damage not for forever (ex. - Ursa's fury swipes stacks). 

(10)
"Vodka" ;D 
(But seriously though , why buy something that gives you so much bad traits. (Or maybe I'm misunderstanding something...))

**Comment by zelding** [comment]
First of all, the numbers are irrelevant. Please check out this document, and preferably update your ideas accordingly [Guidelines - general](../blob/master/doc/general.md)

Secondly, most of your ideas are underpowered, Items don't have to have drawbacks, especially when they provide little benefit to begin with. This might be a side-effect of trying to provide stats, you try to balance them while they are not even in the game.

> Great Titanium Greatsword of Greatness

The high damage with the cost of attack speed is a good idea, but not new. Fatigue as a concept for mechanics is decent.

> Flask of Restoration

Seems kinda unnecessary to me, but could be an optional Bottle upgrade.

> Raging Axe of Throwing

The benefits are minimal for an inventory slot, and you should forget percentage based stats.

> Robin Hood’s Quiver

The whole concept bugs me: quiver for melee heroes, and Robin Hood misses? Also attack speed limit is there for a reason, don't try to mess with it. It can cause laggs and even crashes.

> Pulsar

Seems nice.

> War Banner

I'm not sure if you wanted it to be percentage based or you forgot that attack speed is not just a flat value in dota 2, in the first case they should be just +33 AS; in the second case it would make the item bad.

> HamAxe

Would give permabash, which is unacceptable.

> Slime Shovel of Doom

The 66% regen aura is insane; why not just +10 Hp regen?
The destroyable wall idea is cool. 

> The Death Sabre

Kinda all right, but as @VoidsKeeper already stated it should have a cap.

> The Mad God’s Armour

Joke items are not funny ;)

**Comment by Vafnar** [comment]
Got a break on my phone, Just to clarify a few things, @VoidsKeeper and @zelding when i say -100% magic resistance i mean having 0% magic resistance. as in whatever damage a magical nuke hits you with is the damage you take. When i mean 66% regen rate i am saying, if a hero has 10 base regen then they get another +6 regen. not 66% hpps. I should point out in the document, part of the last item didn't get copied. There should be an active that gives your whole team within 600 units, +15 armour, at the cost of all projectiles coming at you. Also none of them are joke Items (aside from maybe the saxophone) Numbers are there to help with concept.
